src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
The format of this file should be like the existing news article present. Please look at them.